scripts/expire_redshift_data.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+from optparse import OptionParser
+from datetime import date, timedelta
+
+
+USAGE = """Usage:
+        # delete all the rows that are more than 7 days old
+        %prog -n 7
+
+        # delete with specific tables
+        %prog -n 30 -t the_table_name
+        """
+
+_DEFAULT_RETENTION_DAYS = 365 + 31
+_TABLES = ["impression_stats_daily", "application_stats_daily",
+           "newtab_stats_daily", "site_stats_daily"]
+_TEMPLATE = "DELETE FROM {table} WHERE date < {date_lower};"
+_TEMPLATE_VACUUM = "VACUUM {table};"
+_TEMPLATE_ANALYZE = "ANALYZE {table};"
+
+
+def main():
+    parser = OptionParser(usage=USAGE)
+    parser.add_option("-n", "--days",
```

Comment:
This option is dangerous.  Let's remove the '-n' optional altogether and change '--days' to '--days-to-retain'